sw/device/lib/BUILD

```diff
@@ -0,0 +1,49 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package(default_visibility = ["//visibility:public"])
+
+load("//rules:opentitan.bzl", "OPENTITAN_CPU")
+
+cc_library(
```

Comment:
Can you add a comment here indicating that this target is deprecated?

Reply:
We should try the `deprecation` attribute:
https://docs.bazel.build/versions/main/be/common-definitions.html#common-attributes

Reply:
Is there a source I can read/point to for the deprecation?

Reply:
Deprecation of this file, specifically? I don't think so. These drivers are completely un-qualed and were in the process of being replaced with the DIFs in all in-tree uses, but I just haven't had the time. I would simply make the deprecation message be "use the DIF instead" or something.

Reply:
Lets add a deprecation section to the document [The Big Picture for OT's Core Libs](https://docs.google.com/document/d/1L9rCBee-7NW4zw7Pas5euHUudCXHQOoaEkpNvoCEEg8/edit#heading=h.q89cylmz6ddx).

Reply:
I've excerpted the portion that describes the deprecation of these libs.